src/tools/r2rdump/R2RDump.cs

```diff
@@ -11,6 +11,37 @@ namespace R2RDump
 {
     class R2RDump
     {
+        class R2RWriter
```

Comment:
Sorry for the back and forth, but this change made me realize that we may not need a new class for this. We just need to have a TextWriter object that the R2RDump knows about and can make Write and WriteLine calls to. Would you mind making that change?